realm/realm-library/src/main/cpp/io_realm_SyncManager.cpp

```diff
@@ -41,34 +41,35 @@ using namespace realm::jni_util;
 
 std::unique_ptr<Client> sync_client;
 
-JNIEXPORT void JNICALL Java_io_realm_SyncManager_nativeInitializeSyncClient
-    (JNIEnv *env, jclass)
+JNIEXPORT void JNICALL Java_io_realm_SyncManager_nativeInitializeSyncClient(JNIEnv* env, jclass)
 {
     TR_ENTER()
-    if (sync_client) return;
+    if (sync_client)
+        return;
```

Comment:
I think we should add curly brace around single line block if we give it independent line.
@realm/java What do you think?